src/ConnectionConfig.tsx

```diff
@@ -72,21 +80,29 @@ export const ConnectionConfig: FC<ConnectionConfigProps> = (props: ConnectionCon
       <InlineField
         label="Authentication Provider"
         labelWidth={labelWidth}
-        tooltip="Specify which AWS credentials chain to use."
+        tooltip={`Specify which AWS credentials chain to use.${
+          tempCredsFeatureEnabled && awsAllowedAuthProviders.includes(AwsAuthType.GrafanaAssumeRole)
+            ? 'Note: Grafana Assume Role Authentication is only available to Grafana Cloud customers'
```

Comment:
Not sure if this addition is necessary, it was an alternative to checking what kind of an account the user has (not even sure if this is possible), but I can look into it

Reply:
I don't know that I'd add that right now. We set the awsAllowedAuthProviders differently in grafana cloud vs other environments, and we don't really want users to be using this feature yet as it's still under development/experimental. 

I could also see us finding a way to make this feature work in other environments in the future.